src/vmm/tests/integration_tests.rs

```diff
@@ -566,3 +534,62 @@ fn test_snapshot_cpu_vendor_missing() {
     // This must fail as the cpu vendor entry does not exist.
     assert!(validate_x86_64_cpu_vendor(&microvm_state).is_err());
 }
+
+#[cfg(target_arch = "aarch64")]
+#[test]
+fn test_snapshot_cpu_manufacturer_id() {
```

Comment:
Can we have these tests named uniformly for both archs ?

Reply:
Sure